jenkins/release.jenkins

```diff
@@ -39,7 +42,8 @@ pipeline {
       }
       steps {
         sh "docker cp $BUILD_TAG:/code/build ."
-        sh "./bin/aws s3 sync --content-type 'text/html' build/ s3://${params.BUCKET}/${params.RELEASE_ID}/rex/releases"
+        sh "./bin/aws s3 sync --exclude 'books/*' build/ s3://${params.BUCKET}/rex/releases/${params.RELEASE_ID}"
+        sh "./bin/aws s3 sync --content-type 'text/html' build/books/ s3://${params.BUCKET}/rex/releases/${params.RELEASE_ID}/books"
```

Comment:
this is an unrelated change to stop setting the mime type to html for all files, the sync command will infer the mime type from the extension for other files, but the book pages don't have extensions